ludwig/features/base_feature.py

```diff
@@ -352,7 +353,10 @@ def get_metrics(self):
             try:
                 metric_vals[metric_name] = get_scalar_from_ludwig_metric(metric_fn)
             except Exception as e:
-                logger.error(f"Caught exception computing metric: {metric_name}. Exception: {e}")
+                logger.error(
```

Comment:
This should just be:

```
logger.exception(f"Caught exception computing metric: {metric_name}")
```

Then you get the stack trace for free.

Reply:
`logger.exception != logger.error`

Reply:
Ah, didn't catch that. Okay will try that. Thanks!